src/Authentication/Authentication/custom/common/MgCommandMetadata.json

```diff
@@ -95132,7 +95132,7 @@
   },
   {
     "Module": "Beta.Identity.SignIns",
-    "Uri": "/identity/authenticationEventsFlows/{authenticationEventsFlow-id}/onAuthenticationMethodLoadStartExternalUsersSelfServiceSignUp/identityProviders",
+    "Uri": "/identity/authenticationEventsFlows/{authenticationEventsFlow-id}/externalUsersSelfServiceSignUpEventsFlow/onAuthenticationMethodLoadStart/onAuthenticationMethodLoadStartExternalUsersSelfServiceSignUp/identityProviders",
```

Comment:
Please add Pester tests at https://github.com/microsoftgraph/msgraph-sdk-powershell/blob/dev/src/Authentication/Authentication/test/Find-MgGraphCommand.Tests.ps1 to validate the matching of the paths using `Find-MgGraphCommand`.